crypto/fipsmodule/ec/ec_key.c

```diff
@@ -518,7 +526,12 @@ int EC_KEY_generate_key_fips(EC_KEY *eckey) {
   ec_wrapped_scalar_free(eckey->priv_key);
   eckey->pub_key = NULL;
   eckey->priv_key = NULL;
+
+#if defined(BORINGSSL_FIPS)
```

Comment:
I would prefer `AWSLC_FIPS`. But we can make this change globally at some point.